tests/behat/features/admin/report_notification.feature

```diff
@@ -6,10 +6,10 @@ Feature: Admin Report Notification
 
   Scenario: Email subscribed admins directly after report
     Given there are users:
-      | name     | email           | id |
-      | Catrobat | admin@catrob.at |  1 |
-      | User1    | dog@catrob.at   |  2 |
-      | User2    | dog2@catrob.at  |  3 |
+      | name     | email           | id | password |
```

Comment:
I see this test is now failing...

Pls disable the test  @disabled @wip at the top

And create a new Ticket called:

- Report/Upload Notificator stating that those 2 actions in the past should have resulted in an email to the user
I don't think this is needed anymore since it never worked but we have to talk to the POs about this

![image](https://user-images.githubusercontent.com/40868718/80801162-d35d5880-8bab-11ea-8e01-3bb9862cb565.png)